export/index.html

```diff
@@ -565,10 +578,6 @@ <h2>Message log</h2>
   <script type="module">
     // TODO: this should be bundled at build time or replaced with code written by Turnkey entirely.
     import * as hpke from "https://esm.sh/@hpke/core";
```

Comment:
out of scope for this PR but I'd assume we'll want to tackle this as well at some point?

Reply:
yep! this one's a big one so deserves a separate PR. also given that it's used by auth, recovery, export, and import, might want to reorganize the code a bit while we do it so there's less duplication